src/configs/nuage/vsd/index.js

```diff
@@ -40,8 +44,8 @@ const getHeaders = (params = {}) => {
     if (params.page || params.page === 0)
         headers["X-Nuage-Page"] = params.page
 
-    if (params.pageSize)
-        headers["x-nuage-pagesize"] = params.pageSize
+    if (params.size)
```

Comment:
I prefer to keep pageSize for both elastic and vsd! pagination is a common terminology well known. size is too generic